server/xpub-server/entities/manuscript/resolvers/updateManuscript.js

```diff
@@ -22,18 +22,12 @@ async function updateManuscript(_, { data }, { user }) {
   manuscript.applyInput(data)
 
   const newAuthors = manuscript.getAuthor()
-  const newAuthorEmails = newAuthors
-    .map(author => author.alias.email)
-    .join(',')
+  const newAuthorEmails = newAuthors.map(author => author.alias.email).join(',')
```

Comment:
not sure why the commit hooks reformatted this, but it's identical to before in functionality